### ISSUE TITLE
개념증명 실험

### ISSUE THREAD

**Issue by alwaysY0ung** [alwaysY0ung/Deep-learning-based-DGA#1]
##### 1차 개발 3d1ef21
Question: "I am curious why the images from a well-performing, deceptive attack model don't look like a '5' to the human eye."
Feedback: "It's because the loss function only measures the model's numerical performance at deceiving the classifier (using Binary Cross Entropy), and not its human-perceivable appearance."
